lib/project_types/script/messages/messages.rb

```diff
@@ -50,6 +50,25 @@ module Messages
 
           script_not_found_cause: "Couldn't find script %s for extension point %s",
 
+          service_failure_cause: "Internal service error. " \
+                                 "Ensure the 'shopify/scripts-toolchain-as' package is up to date.",
```

Comment:
Sorry, I should've clarified, the `error_handler` file has an optional `help_suggestion` key in the hash returned for a given error adjacent to the `cause_of_error` key. The help suggestions should be provided with that key so they're rendered appropriately.